addon/components/sl-modal.js

```diff
@@ -0,0 +1,236 @@
+import Ember from 'ember';
+import layout from '../templates/components/sl-modal';
+
+/**
+ * @module
+ */
+export default Ember.Component.extend({
+    // -----------------------------------------------------
+    // Dependencies
+
+
+    // -----------------------------------------------------
+    // Attributes
+
+
+    /** @type {String[]} */
+    attributeBindings: [
+        'aria-describedby',
+        'aria-hidden',
+        'aria-labelledby',
+        'tabindex'
+    ],
+
+    /** @type {String[]} */
+    classNames: [
+        'modal fade'
+    ],
+
```

Comment:
Previous implementation had classNameBindings of 'animated:fade'   Are we still supporting the same UI capabilities as before?